data/formats-data.js

```diff
@@ -6360,4 +6360,94 @@ exports.BattleFormatsData = {
 		isNonstandard: true,
 		tier: "CAP",
 	},
+ +	brycenman: {
+ +		randomBattleMoves: ["icebeam", "nightshade", "psychic", "uturn"],
+ +		isNonstandard: true,
+		eventOnly: true,
```

Comment:
I suppose so. I added the flag because the only moves in their learnsets come from events, and because they don't have shiny sprites, but if they're only usable in BH and Custom Game, that flag won't matter. I'll remove those when I add their "events" to this file.